recipes/irrxml/all/patches/1.2/irrtypes_debug_vs.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/src/irrTypes.h b/src/irrTypes.h
+index aa2254d..8296b8d 100644
+--- a/src/irrTypes.h
++++ b/src/irrTypes.h
+@@ -79,11 +79,7 @@ typedef unsigned short wchar_t;
+ #endif // microsoft compiler
+ 
+ //! define a break macro for debugging only in Win32 mode.
+-#if defined(WIN32) && defined(_MSC_VER) && defined(_DEBUG)
+-#define _IRR_DEBUG_BREAK_IF( _CONDITION_ ) if (_CONDITION_) {_asm int 3}

```

Comment:
probably, it should be **DebugBreak()** call here

Reply:
I gonna remove this patch. I've checked other package managers, and only we are using it.